src/global/global_init.cc

```diff
@@ -294,6 +295,7 @@ void global_init_daemonize(CephContext *cct)
 
   global_init_postfork_start(cct);
   global_init_postfork_finish(cct);
+#endif
```

Comment:
I've added a warning instead as the current dependency/include structure requires it's inclusion. The method global_init_daemonize is only used by:

./ceph_mds.cc
./ceph_osd.cc
./global/global_init.cc
./rgw/rgw_main.cc
./rgw/rgw_object_expirer.cc
./test/messenger/simple_server.cc
./test/messenger/xio_server.cc